notify/src/lib.rs

```diff
@@ -173,22 +238,43 @@ impl NotifyService {
         let _ = responder.send(receiver);
     }
 
-    async fn handle_notify_new_block(&mut self, block: BlockView) {
+    fn handle_notify_new_block(&self, block: BlockView) {
         trace!("event new block {:?}", block);
+        let block_hash = block.hash();
         // notify all subscribers
-        for subscriber in self.new_block_subscribers.values() {
-            let _ = subscriber.send(block.clone()).await;
+        for subscriber in self.new_block_subscribers.values().cloned() {
+            let block = block.clone();
+            self.handle.spawn(async move {
+                if let Err(e) = subscriber.send(block).await {
+                    error!("notify new block error {}", e);
+                }
+            });
```

Comment:
Since rust lacks the sugar `clone_to_closure`, we can only clone var manually, but since we have already cloned the `block`, I suggest cloning the `subscriber` again, which will remove the `clippy::unnecessary_to_owned` hint also:

```suggestion
        for subscriber in self.new_block_subscribers.values() {
            let subscriber = subscriber.clone();
            let block = block.clone();
            self.handle.spawn(async move {
                if let Err(e) = subscriber.send(block).await {
                    error!("notify new block error {}", e);
                }
            });
        }
```